Source/Modules/csharp.cxx

```diff
@@ -254,6 +256,9 @@ class CSHARP:public Language {
 	} else if (strcmp(argv[i], "-oldvarnames") == 0) {
 	  Swig_mark_arg(i);
 	  old_variable_names = true;
+	} else if (strcmp(argv[i], "-enablenullable") == 0) {
```

Comment:
Please document this new option in CSharp.html, including a sentence or two as to why it is needed (for those who have not heard of the preprocessor directive) and a link to the MS docs.